Tutorials/Tutorial25_StatePackager/assets/path_trace.csh

```diff
@@ -9,7 +9,8 @@ Texture2D g_Normal;
 Texture2D g_Emittance;
 Texture2D g_Depth;
 
-RWTexture2D<float4 /*format = rgba32f*/> g_Radiance;
+Texture2D<float4>                        g_RadianceInput;
+RWTexture2D<float4 /*format = rgba32f*/> g_RadianceOutput;
```

Comment:
Please only use this on WebGPU.
On anything else, there should be just one texture.